CMakeLists.txt

```diff
@@ -6,7 +6,8 @@ set(CMAKE_CXX_STANDARD 17)
 
 set(LIB ${CMAKE_SOURCE_DIR}/lib)
 add_subdirectory(${LIB}/spdlog)
-
+add_subdirectory(${LIB}/eigen)
+add_subdirectory(${LIB}/rapidcsv)
 
 
 add_executable(ISS_SILSIM src/main.cpp
```

Comment:
RASAeroImort.cpp needs to be added to the sources in the add_executable call

Reply:
Fixed